nifi-provenance-reporting-tasks/src/main/java/io/egm/nifi/reporting/AbstractProvenanceReporter.java

```diff
@@ -80,6 +80,18 @@ public abstract class AbstractProvenanceReporter extends AbstractReportingTask {
             .defaultValue("https://localhost:443")
             .addValidator(StandardValidators.URL_VALIDATOR).build();
 
+    static final PropertyDescriptor CHECK_FOR_HTTP_ERRORS = new PropertyDescriptor.Builder().name("check-for-http-errors")
```

Comment:
you can also set allowed values like in https://github.com/apache/nifi/blob/2559fc1e53242f03f263e5e68003342c8705bb03/nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java#L215. it helps to avoid mistakes.